osv/repos.py

```diff
@@ -131,7 +131,9 @@ def _use_existing_checkout(git_url,
   repo = pygit2.Repository(checkout_dir)
   repo.cache = {}
   if repo.remotes['origin'].url != _git_mirror(git_url):
-    raise RuntimeError('Repo URL changed.')
+    logging.warn('origin url updated:\nOld: %s\nNew: %s',
```

Comment:
Nit: uppercase URL

Reply:
Please also update the description to capture why you're making this URL change.